pkg/resource/aws/aws_iam_access_key_test.go

```diff
@@ -0,0 +1,44 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+func TestAwsIamAccessKey_String(t *testing.T) {
+	tests := []struct {
+		user   string
+		access AwsIamAccessKey
+		want   string
+	}{
+		{user: "test iam access key stringer with user and id",
+			access: AwsIamAccessKey{
+				User: aws.String("test_user"),
+				Id:   "AKIA2SIQ53JH4CMB42VB",
+			},
+			want: "AKIA2SIQ53JH4CMB42VB (User: test_user)",
+		},
+		{user: "test iam access key stringer without user",
+			access: AwsIamAccessKey{
+				User: nil,
+				Id:   "AKIA2SIQ53JH4CMB42VB",
+			},
+			want: "AKIA2SIQ53JH4CMB42VB",
+		},
+		{user: "test iam access key stringer with empty user",
```

Comment:
You can remove this test as it does not have sense anymore

Reply:
Ok, thank you